src/models/OtrStore.js

```diff
@@ -53,6 +56,34 @@ OtrStore.prototype.initChan = function(args) {
 			session.on('io', function(msg, meta) {
 				irc.send(chan.name, msg);
 			});
+
+			session.on('error', function(err, severity)  {
+				// severity is one of "warning" or "error"
+				var body = {type: severity, msg: err};
```

Comment:
Not my favorite, but the style chosen across the entire project seems to be one line per attribute, even for one-liners. Considering this, and that you're using `body` exactly once, and calling the next body `payload`, I would go with:

```js
client.emit('otr', {
	type: severity,
	msg: err
});
```